src/modules/esl-media-query/test/esl-media-rule-list.test.ts

```diff
@@ -178,4 +178,25 @@ describe('ESLMediaRuleList', () => {
       expect(() => ESLMediaRuleList.parseTuple('@xs', '1|2|3')).toThrowError();
     });
   });
+
+  describe('Mixed cases parsing', () => {
```

Comment:
Ideally is to focus on different input options here rather than on checking than check that media work (which is other test scope of work)

Please provide more cases. Id suggest the following at leat:


| 1-st arg | 2-nd arg | Result (we can use `toString` to check that everything parsed  correct) |
| --- | --- | --- |
| `123` | - | `all => 123` | 
| `(min-width: 100px) => 123` | - | `(min-width: 100px) => 123` |
| `0 \| (min-width: 100px) => 1 \|  (min-width: 200px) => 2` | - | `all => 0 \| (min-width: 100px) => 1 \|  (min-width: 200px) => 2` | 
| `0 \| (min-width: 100px) => 1 \|  (min-width: 200px) => 2` | `@xs \| @sm` | `all => 0 \| (min-width: 100px) => 1 \|  (min-width: 200px) => 2` | 
| `0 \| 1 ` | `@+xs \| @+sm` | ` (min-width: ...px) => 1 \| (min-width: ...px) => 2` |